page_objects/page_object_model.js

```diff
@@ -0,0 +1,36 @@
+var PageObject = function () {}
```

Comment:
Can you try to refactor this class ?

Idea of POM is to have separated class for every page, for example for log in page you can have something like this:

class LoginPage  {

//GETTERS
//here you put all locators for log in, for example email fields, password field, login button etc


//ACTIONS
//Here you put basic action related to getters above.
For example method "enterEmail", "enterPassword", "ClickLogIn", "DeletePassword", "ClickCancel". etc
}

And then, inside your Smoke test you just make call to these methods.
Something like this inside Smoke test:

LoginPage.enterEmail(data.email)
                  .enterPassword(data. password)
                  .clickLogIn().. etc

Reply:
https://www.toolsqa.com/selenium-webdriver/page-object-model/
useful link to see this concept